include/app_cert.h

```diff
@@ -0,0 +1,38 @@
+/* Sockets Example
+ * Copyright (c) 2020 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+static const char app_cert[] = \
```

Comment:
`app_cert` looks misleading. This seems to be used as the root CA for the TLS connection (what the local TLS stack will use to validate the remote cert).

Reply:
what name should it have do you think?